source/jormungandr/jormungandr/realtime_schedule/siri.py

```diff
@@ -177,8 +233,11 @@ def _get_passages(self, xml, route_point):
             cur_route = visit.find('.//siri:DirectionName', ns).text
             if route != cur_route:
                 continue
+            # TODO? we should ignore MonitoredCall with a DepartureStatus set to "Cancelled"
```

Comment:
How important are those TODOs ? Shall we create a Jira about it to keep track of them ?

Reply:
As stated in the PR message those are proposition of improvement, I think we should do them but in another PR